core/templates/pages/contributor-dashboard-page/opportunities-list/opportunities-list.component.ts

```diff
@@ -45,8 +45,10 @@ export class OpportunitiesListComponent {
   @Input() opportunityHeadingTruncationLength: number;
   @Input() opportunityType: string;
 
+  private static readonly MAX_PAGE_NUMBER = 1000;
```

Comment:
I don't understand what this means, sorry. Might need revised naming.

Is it the maximum possible page number that can be shown in the frontend (i.e. if there are >= 1000 pages we can't show page 1001)?

Or is it the current max page number that's being shown? In that case it seems weird to me that lastPageNumber is 1000 because that implies to me that there are 1000 pages of results whereas in reality there may be fewer.